Contributing.md

```diff
@@ -58,7 +58,7 @@ We'll close your PR or issue if:
 1. It's a duplicate of an existing issue.
 2. Outside of the scope of the project.
 3. The bug is not reproducible.
-4. You are unresponsive after a few days.
+4. You don't response after a few days.
```

Comment:
it would be "respond" and means the same

Reply:
Shame on me. And thank you very much